anvio/docs/artifacts/aa-frequencies-txt.md

```diff
@@ -0,0 +1,3 @@
+This file contains **the frequency of each amino acid in each gene in your %(contigs-db)s.** 
```

Comment:
I would suggest that any artifact describing text files (especially input text files) includes an example of just a few lines in the file, since it is easier and faster to understand the format by looking at it rather than reading the description.

It is not really necessary for output files, but could still be useful so that people understand what to expect (and perhaps plan how they might use the file downstream).